frequency: 30.0
sensor_timeout: 0.1
two_d_mode: true
publish_tf: true

imu0:   /imu/data
imu0_config:  [false,false,false,  false,false,false,  true,true,true,  true,true,true]

odom0:  /odom
odom0_config: [true,true,false,   false,false,false,  true,false,false, false,false,false]

base_link_frame: base_link
world_frame:     odom
odom_frame:      odom
